mteb/models/vdr_models.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from functools import partial
+
+import torch
+
+from mteb.encoder_interface import PromptType
+from mteb.model_meta import ModelMeta
+from mteb.models.instruct_wrapper import InstructSentenceTransformerWrapper
+
+
+def instruction_template(
+    instruction: str, prompt_type: PromptType | None = None
+) -> str:
+    return "<|im_start|>system\nYou are a helpful assistant.<|im_end|>\n<|im_start|>user\n%s<|im_end|>\n<|endoftext|>"
```

Comment:
I don't think you need to specify this, because this will be applied in their implementation https://huggingface.co/llamaindex/vdr-2b-multi-v1/blob/2c4e54c8db4071cc61fc3c62f4490124e40c37db/custom_st.py#L234
But in that case instructions will be applied in user prompt, which can be incorrect, but I'm not sure how to do it right. Maybe @logan-markewich or @marplex can suggest something

Reply:
cool, I will wait for their reply.

Reply:
@Samoed, should we close this now with this instruction_template? Then you can look at it once again when you implement the multimodal input for it.

Reply:
You should change template to apply instruction. Other template will be added inside sentence transformers automatically
```suggestion
    return "{instruction}"
```

Reply:
@Samoed done, this one